ingestion/src/metadata/ingestion/source/database/glue.py

```diff
@@ -73,20 +66,6 @@ def __init__(self, config: WorkflowSource, metadata_config: OpenMetadataConnecti
         )
         self.task_id_mapping = {}
```

Comment:
maybe we can remove this variable from here too

Reply:
Sure.